Assignment03.Tests/ExtensionsTest.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using Xunit;
+
+namespace Assignment03.Tests
+{
+  public class ExtensionsTests
+  {
+
+  }
+}
```

Comment:
If this class isn't used, please delete. We can move the Extensions tests into this class, but I don't see a reason for doing it.

Reply:
It is just being stated that: "Use the supplied Extensions.cs and ExtensionsTests.cs files.". And this empty file is already a part of their repository. I just didn't feel like moving all the tests into it on this commit, since it should be pushed with the rest of the extension methods. I just considered that if we have the file already in master, we are less likely to forget about it :D

Reply:
I don't see how that justifies its existance? If we don't intend on using it, we should remove it. We use other classes to do what it would have done, which justifies the removal of it, if anything.